rust/core-lib/src/rpc.rs

```diff
@@ -175,12 +175,19 @@ pub enum CoreNotification {
     /// ```
     Plugin(PluginNotification),
     /// Tells `xi-core` to close the specified view.
-    CloseView { view_id: ViewId },
+    CloseView {
```

Comment:
I think these changes are unintended?

Reply:
@sjoshid this is still unresolved

Reply:
I see similar changes for other enum types. I think `rustfmt`  did this but if not, I can revert these changes.

Reply:
I just confirmed. rustfmt did this change.